integrations/opensearch/src/haystack_integrations/document_stores/opensearch/document_store.py

```diff
@@ -174,6 +201,7 @@ def write_documents(self, documents: List[Document], policy: DuplicatePolicy = D
             refresh="wait_for",
             index=self._index,
             raise_on_error=False,
+            max_chunk_bytes=self._max_chunk_bytes,
```

Comment:
`bulk` is also used in `delete_documents`. Can you please add `max_chunk_bytes` there for consistency too?